sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -604,6 +611,20 @@ private void updateListOfDays(Integer selectedYear, Integer selectedMonth) {
 		}
 	}
 
+	private void initializePresentConditionField() {
+		PresentCondition presentCondition = getValue().getPresentCondition();
+		ComboBox presentConditionField = getField(PersonDto.PRESENT_CONDITION);
+
+		/*
+		 * It may happen that the person currently has a present condition that usually shall not be shows for the form's disease.
+		 * In that case, the present condition is added as selectable item here.
+		 */
+		if (presentConditionField.getItem(presentCondition) == null) {
+			Item currentItem = presentConditionField.addItem(presentCondition);
+			currentItem.getItemProperty(SormasFieldGroupFieldFactory.CAPTION_PROPERTY_ID).setValue(presentCondition.toString());
```

Comment:
if the `presentCondition` is null, this will result in a NPE. It just happened on my systems when I tried opening an existing Person

Reply:
Even though I also noticed that problem, I forgot to add a null check here. Sorry.
There's a fix committed now.